src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -49,6 +49,10 @@
     private $filterCollection;
     private $nameConverter;
 
+    private $mimeTypes;
```

Comment:
Can you get rid of those states please? A service should not have states like those ones.